python/repositories.bzl

```diff
@@ -183,7 +183,6 @@ python_repository = repository_rule(
         "python_version": attr.string(
             doc = "The Python version.",
             mandatory = True,
-            values = TOOL_VERSIONS.keys() + MINOR_MAPPING.keys(),
```

Comment:
I think there should still be some validation here, but instead now in the macro. Can this be added in a follow up diff?